Xplat/src/main/resources/assets/neat/lang/pt_br.json

```
@@ -0,0 +1,4 @@
+{
+    "neat.keybind.toggle": "Alternar barras de saúde"
+  }

```

can you delete the extra spaces at the start of this line?
```suggestion
}

```